src/wasm/extract.cpp

```diff
@@ -48,13 +49,15 @@ file_output::file_output(const fs::path& dir, const processed_file* f, bool writ
   }
 }
 
-bool file_output::write(const char* data, size_t n) {
-  int r;
-  if(!zip_open_) {
-		r = zip_.addFile(path_.c_str(), data, n, &zipindex_);
-		zip_open_ = true;
+bool file_output::write(char* data, size_t n) {
+  ZIPentry *z;
+  if(!file_open_) {
```

Comment:
Missing space before parenthesis.
```suggestion
  if (!file_open_) {
```